types/artifacts.d.ts

```diff
@@ -399,6 +399,10 @@ declare global {
         naturalWidth: number;
         /** The natural height of the underlying image, uses img.naturalHeight. See https://codepen.io/patrickhulce/pen/PXvQbM for examples. */
         naturalHeight: number;
+        /** The attribute width of the image, ... */
```

Comment:
```suggestion
        /** The raw width attribute of the image element. CSS images will be set to the empty string. */
```

maybe?

Reply:
I like that clarification, I'll change it to that